pennylane/gradients/gradient_transform.py

```diff
@@ -214,34 +214,26 @@ def jacobian_wrapper(*args, **kwargs):
 
             qjac = _wrapper(*args, **kwargs)
 
-            if any(m.return_type is qml.operation.Probability for m in qnode.qtape.measurements):
-                qjac = qml.math.safe_squeeze(qjac, exclude_axis=-1)

```

Comment:
wow, really nice that this can be removed!

Reply:
Yes, really happy to see that, too! :)

Reply:
Actually, `safe_squeeze` only was introduced for this functionality, and it seems like it was doing a job that is done with this PR more thoroughly. We could consider removing it altogether, what do you think @josh146 ?

Reply:
I don't mind either way 🙂

Reply:
I removed the function then since it goes unused 👍